tests/VolleyManagement.UnitTests/WebApi/Controllers/ScheduleTests.cs

```diff
@@ -227,12 +238,14 @@ public void GetSchedule_TournamentPlayedOverSeveralYears_ScheduleIsOrderedByYear
         public void GetSchedule_TournamentHasFreeDayGame_FreeDayGameIsLast()
         {
             // Arrange
-            var testTournament = new TournamentBuilder().Build();
-            MockGetTournament(testTournament, TOURNAMENT_ID);
+            const int TEST_ROUND_COUNT = 5;
```

Comment:
byte

Reply:
use byte instead int, nothing personal

Reply:
if you can't explain why it becomes personal